tests/integration/config/test_provider_env.py

```
@@ -0,0 +1,493 @@
+        assert text_cfg.name == "llama3:8b"
+        assert vision_cfg.name == "llava:13b"
+
+
+# ---------------------------------------------------------------------------
+# _get_model_configs_from_profile
+# ---------------------------------------------------------------------------
+
+
+class TestGetModelConfigsFromProfile:
+    def test_returns_none_when_profile_uses_default_preset(
+        self, monkeypatch: pytest.MonkeyPatch
```

Ruff (0.15.9)

[warning] Unused method argument: `monkeypatch`

(ARG002)